lib/src/iterable_extensions.dart

```diff
@@ -590,7 +590,6 @@ extension IterableNumberExtension on Iterable<num> {
       }
       return value;
     }
-    return null;
```

Comment:
Please do not make this change.

I personally think the missing `return null;` at the end should be a compile-time error (and will work towards making it so).

If it's a compile-time error to have a `return;` with no value, it should not be allowed to have an explicit fallthrough.